oh-my-posh.rb

```diff
@@ -11,6 +11,7 @@ class OhMyPosh < Formula
 
   def install
     Dir.chdir("src") do
+      ENV["GOPROXY"] = ENV.has_key?("HOMEBREW_GOPROXY") ? ENV["HOMEBREW_GOPROXY"] : ""
```

Comment:
If you add it here, it also needs a revision in this file specifically (not the template).

Reply:
So should I just leave this unchanged?

Reply:
That's also possible 😬

Reply:
Aha, I didn't realize that I can add a line of `revision 1` until I saw a previous [commit](https://github.com/JanDeDobbeleer/homebrew-oh-my-posh/commit/035acb50184a1df12f01a3861880263f5b1f7510#diff-c46d4ec6ce73e2a6f5a09db055db5b63de724131731c92c9e241e9476462e5daR8). 😆